lua/nvim-treesitter/incremental_selection.lua

```diff
@@ -138,22 +139,19 @@ function M.attach(bufnr)
   local config = configs.get_module "incremental_selection"
   for funcname, mapping in pairs(config.keymaps) do
     if mapping then
-      ---@type string, string|function
-      local mode, rhs
-      if funcname == "init_selection" then
-        mode = "n"
-        ---@type function
-        rhs = M[funcname]
+      local mode = funcname == "init_selection" and "n" or "x"
+      local rhs = M[funcname] ---@type function
+
+      if not rhs then
+        utils.notify("Unknown keybinding: " .. funcname, vim.log.levels.ERROR)
```

Comment:
Not mandatory, but this might be slightly more helpful:

```suggestion
        utils.notify("Unknown keybinding: " .. funcname .. "\n" .. debug.traceback(), vim.log.levels.ERROR)
```